codegen.yml

```diff
@@ -2,10 +2,6 @@ overrideExisting: true
 schema: 'https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v3'
 documents: 'src/**/!(*.d).{ts,tsx}'
 generates:
-  ./src/state/data/generated.ts:
+  ./schema/schema.graphql:
```

Comment:
The old path was intentionally omitted from version control w/ .gitignore

It's generally best-practice to omit generated code from git. You should update the path to ignore the generated files here as well.